### DISCUSSION TITLE
ORVIBO SN10ZW not support CC2652

### DISCUSSION THREAD

**Discussion by qlwz** [arendst/Tasmota#13168] [Q&A] [answered]
CC2652 ：Zigbee2MQTT is normal，Please check the log。
CC2652 ：Tasmota is no Occupancy，Please check the log。
CC2530： Tasmota is normal，Please check the log。

@s-hadinger 

[Zigbee2MQTT Log.txt](https://github.com/arendst/Tasmota/files/7191588/Zigbee2MQTT.Log.txt)

[Tasmota Log CC2530.txt](https://github.com/arendst/Tasmota/files/7191701/Tasmota.Log.CC2530.txt)

[Tasmota Log CC2652.txt](https://github.com/arendst/Tasmota/files/7191702/Tasmota.Log.CC2652.txt)

**Comment by qlwz** [comment]
@s-hadinger can you help me?

**Reply by s-hadinger** [reply]
Maybe but I need to find time to analyze the logs

**Reply by qlwz** [reply]
> Maybe but I need to find time to analyze the logs

Zigbee2MQTT 1.17.0 is normal   https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.17.0
Zigbee2MQTT 1.16.2  is no normal  https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.16.2

**Comment by qlwz** [comment, marked as answer]
@s-hadinger   https://github.com/Koenkk/zigbee-herdsman/commit/77079f11  please add this code
